test/legacy_test/test_generate_proposals_op.py

```diff
@@ -350,7 +350,7 @@ def set_data(self):
 
     def test_check_output(self):
         # NODE(yjjiang11): This op will be deprecated.
-        self.check_output(check_dygraph=False)
+        self.check_output(check_dygraph=False, check_pir=True)
```

Comment:
```suggestion
        self.check_output(check_dygraph=False)
```
当前 `generate_proposals` 应该已经废弃，只使用 `generate_proposals_v2` 了。所以该单测可以不适配